frontend/app/abtests/PayPalTestVariants.scala

```diff
@@ -0,0 +1,10 @@
+package abtests
+
+import scala.util.Random
+
+object PayPalTestVariants {
+  val withPayPal = "with_paypal"
+  val stripeOnly = "stripe_only"
+  val variants = List(withPayPal, stripeOnly)
```

Comment:
If you decide to go with the approach of avoiding stringly-typed stuff, you will have to modify `Binders.scala`. See this PR with the same type of changes: https://github.com/guardian/membership-frontend/pull/1437/files#diff-108bb3217f691dfd764ade3ec6ff3196L1